src/aioauth/grant_type.py

```diff
@@ -68,6 +68,17 @@ async def validate_request(self, request: Request) -> Client:
 
         return client
 
+    def get_client_credentials(
+        self, request: Request
+    ) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
`-> Tuple[str, str]:` should fix the current issue with the auto-checker.

Reply:
Hi, I changed the return type as you suggested. Thank you.
The project should have a type checker in pre-commit-config to catch these problems before opening a PR. Maybe add a mypy.ini and a mypy section to pre-commit-config or a pyright equivalent.